FAQ.md

```diff
@@ -103,7 +103,7 @@ Metrics Server was tested to run within clusters up to 5000 nodes with an averag
 
 #### How often metrics are scraped?
 
-Default 60 seconds, can be changed using `metric-resolution` flag. We are not recommending setting values below 15s, as this is the resolution of metrics calculated by Kubelet.
+Default 10 seconds, can be changed using `metric-resolution` flag. We are not recommending setting values below 10s, as this is the resolution of metrics calculated by Kubelet.
```

Comment:
Default is 60 seconds in code: https://github.com/kubernetes-sigs/metrics-server/blob/master/cmd/metrics-server/app/options/options.go#L105

Also, the fact that we are recommending below 15s is set in stone since this is the value we are running scalability tests with.